pkgs/applications/misc/kratos/default.nix

```diff
@@ -0,0 +1,45 @@
+{ fetchFromGitHub, buildGoModule, lib, stdenv }:
+
+buildGoModule rec {
+  pname = "kratos";
+  version = "0.6.0-alpha.1.pre.2";
+
+  src = fetchFromGitHub {
+    owner = "ory";
+    repo = "kratos";
+    rev = "v${version}";
+    sha256 = "1jyhb5cs532msqfgnhn6bnnqw6d94g3pjgirklbrysd3z4f32f8g";
+  };
+
+  vendorSha256 = "1n5g6mrr5c8z6y3ncpkap971n6m5qimpfqkq8vjgl8jkih4m282x";
+
+  subPackages = [ "." ];
+
+  buildInputs = [ stdenv ];
```

Comment:
I think this is not needed.

Reply:
I've removed the buildInputs, but removing subPackages triggers the following error:

`main module (github.com/ory/kratos) does not contain package github.com/ory/kratos/corp`

I've tried messing with go.mod, but didn't manage to fix this.

Reply:
`subPackages` is ok, i only meant `buildInputs`, sorry